pkg/apis/autoscaling/v1/clusterautoscaler_types.go

```diff
@@ -8,6 +8,17 @@ func init() {
 	SchemeBuilder.Register(&ClusterAutoscaler{}, &ClusterAutoscalerList{})
 }
 
+// ExpanderString contains the name of an expander to be used by the cluster autoscaler.
+// +kubebuilder:validation:Enum=least-waste;priority;random
```

Comment:
Enums should always use PascalCase and convert inside the operator to the upstream equivalent.
```suggestion
// +kubebuilder:validation:Enum=LeastWaste;Priority;Random
```